pkg/controller/accountclaim/accountclaim_finalizer_test.go

```diff
@@ -0,0 +1,242 @@
+package accountclaim
```

Comment:
IMHO unit tests shouldn't cover package-internal functions but only exported functions (the public interface). This can be easily enforced by putting the tests in a different package than the functions we test, like `accountclaim_test`